src/strategies/spidStrategy.ts

```diff
@@ -129,6 +143,30 @@ const spidStrategy = (
     }
   };
 
+  const IDP_IDS: { [key: string]: string } = {
+    [ARUBA_ID]: "arubaid",
+    [INFOCERT_ID]: "infocertid",
+    [INTESA_ID]: "intesaid",
+    [LEPIDA_ID]: "lepidaid",
+    [NAMIRIAL_ID]: "namirialid",
+    [POSTE_ID]: "posteid",
+    [SIELTE_ID]: "sielteid",
+    [SPIDITALIA_ID]: "spiditalia",
+    [TIM_ID]: "timid"
+  };
+
+  if (!process.env.IDP_METADATA_URL) {
```

Comment:
there is already a mechanism in this codebase to require an env var, see [here](https://github.com/teamdigitale/io-backend/blob/master/src/container.ts#L172)

Reply:
fixed in commit ebb3cf5